data/EternalCoderss.json

```diff
@@ -41,7 +41,7 @@
     },
     {
       "name": "Hashnode Blog",
-      "url": "hhttps://hashnode.com/@shivi",
+      "url": "hhttps://hashnode.com/shivi",
```

Comment:
there's still a extra `h` in the https

it should be `https://hashnode.com/@shivi`

Reply:
keep the `@`